tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -303,10 +303,26 @@
       - openstack
     EOF
 
-- name: wait for dataplane node set to be ready
-  ansible.builtin.shell: |
-    {{ shell_header }}
-    {{ oc_header }}
-    oc wait --for condition=Ready osdpns/openstack --timeout=40m
+- name: Validate EDPM deployment
   # TODO: work on network configuration for making possible to run this task on other IP ranges
   when: "edpm_node_ip.startswith('192.168.122')"
+  block:
+    - name: wait for dataplane node set to be ready
+      ansible.builtin.shell: |
+        {{ shell_header }}
+        {{ oc_header }}
+        oc wait --for condition=Ready osdpns/openstack --timeout=40m
+
+    - name: trigger manual compute host discovery in nova
+      ansible.builtin.shell: |
+        {{ shell_header }}
+        {{ oc_header }}
+        oc rsh nova-cell0-conductor-0 nova-manage cell_v2 discover_hosts --verbose
+
+    - name: run make edpm_deploy_instance
+      ansible.builtin.shell: |
+        {{ shell_header }}
+        {{ oc_header }}
+        cd {{ install_yamls_path }}
+        cd devsetup
+        make edpm_deploy_instance
```

Comment:
@bogdando has a patch that boots an instance on the 17.1 side before the adoption and checks that that instance is still visible and manageable after the adoption[1]. So you might want to sync with @bogdando about this.

Also in general I vote to run a small tempest smoke test instead of using this make target. But that can be a follow up PR for sure.

[1] https://github.com/openstack-k8s-operators/data-plane-adoption/pull/193/files#diff-82241d5b060f40b8d4887dd11209cc9a576cbe96a77bcb324a9f911f64e43145R450

Reply:
please note that the linked checks only target workloads f"rom the Wallaby cluster
to be migrated", i.e. created before any service adoption starts. We might want to boot another instance here as well, now using the Anteleope code

Also, this task "Validate EDPM deployment" is missing in the corresponding documentation sections